frontend/leads-list/table/lead-services-cell.component.tsx

```diff
@@ -0,0 +1,274 @@
+                alt="immigration-icon"
+                src={immigrationIcon}
+                title="Immigration"
+              />
+              <span>( {immigrationServices.length} )</span>
+            </div>
+          )}
+          {financialServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="financial-icon"
+                src={financeIcon}
+                title="Financial Education / Coaching"
+              />
+              <span>( {financialServices.length} )</span>
+            </div>
+          )}
+          {workersRightsServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="workers-rights-icon"
+                src={workerRightsIcon}
+                title="Workers' Rights"
+              />
+              <span>( {workersRightsServices.length} )</span>
+            </div>
+          )}
+          {familySupportServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="family-services-icon"
+                src={familySupportIcon}
+                title="Family Support"
+              />
+              <span>( {familySupportServices.length} )</span>
+            </div>
+          )}
+          {communityEngagementServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="community-engagement-icon"
+                src={communityEngagementIcon}
+                title="Community Engagement and Organizing"
+              />
+              <span>( {communityEngagementServices.length} )</span>
+            </div>
+          )}
+          {focusGroupServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="focus-groups-icon"
+                src={focusGroupsIcon}
+                title="Focus Group"
+              />
+              <span>( {focusGroupServices.length} )</span>
+            </div>
+          )}
+        </div>
+      ) : (
+        <p>No services selected</p>
+      )}
+    </>
+  );
+}
+
+const css = `
+  & .programs-cell {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    font-size: 1.25rem;
+    font-weight: 600;
+  }
+  
+  & .icon-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0.5rem;
+  }
+  
+  & .program-icon {
+    height: 2rem;
+    margin-right: 0.35rem;
+  }
+`;
+
+type LeadServicesCellProps = {
+  leadServices: Array<LeadService>;
+};
+
+type LeadService = {
+  id?: number;
+  serviceName?: string;
+  programId?: number;
+  programName?: string;
+};
+
+type HealthService = {
```

Comment:
All these service types are the same - no need to redefine the same typescript type many times